advocacy_docs/kubernetes/cloud_native_postgresql/samples/cluster-example-initdb.yaml

```diff
@@ -9,8 +9,11 @@ spec:
     initdb:
       database: appdb
       owner: appuser
+      postInitSQL:
+        - create table numbers (i integer)
+        - insert into numbers (select generate_series(1,10000))
       options:
       - "-k"
-      - "--locale=en_US"
+      - "--locale=en_ZA.utf8"
```

Comment:
This seems odd. For reference, en_ZA is [English South Africa](https://metacpan.org/dist/DateTime-Locale/view/lib/DateTime/Locale/en_ZA.pod). I doubt it's a problem, but I'm curious why it was changed to that.

Reply:
Huh... That originated with this PR, might wanna drop a question in: https://github.com/EnterpriseDB/cloud-native-postgresql/pull/593